### PR TITLE
Rnn with attn

### PR DESCRIPTION
**Pull request by Prajakta15Patil**
Can you please review my code covid_clustering.ipynb and covid_bert.ipynb? I have added markdowns.

### CODE REVIEW

File: code/extract_data.py [hunk 1]
```diff
@@ -58,20 +58,21 @@ def process_paper(paper):
 
 # print(len(data)) # 19458
```
Comment (upomodoro) [inline code/extract_data.py]: remove comment.

File: code/extract_data.py [hunk 2]
```diff
@@ -58,20 +58,21 @@ def process_paper(paper):
...
 
-
+# don't export all files into one large file. break it up.
 def exportdata(data, path):
```
Comment (upomodoro) [inline code/extract_data.py:97]: Function should have docstring

**Review by upomodoro — CHANGES_REQUESTED** [review]
Hey Prajakta,

Thanks for putting the pull request together. There a few things you should do, but the biggest thing you can do is annotate your code some more and delete any files that are not necessary for the code to run.

1. DailyMail-Bert is just one cell?

2. What is the vis.html file?

3. Do you need code/rnn_seq2seqattn_upom.ipynb, rnn_seq2seqattn_cnn_dailymail.ipynb

4. All functions in extract_data.py need docstrings.

5. Covid-BERT.ipynb has a lot of output. It would be ideal if it was reduced.

6. covid_analysis.ipynb needs a summary and annotation

7. RNN_LSTMwithAttn.ipynb, RNN_covid.ipynb, and RNN_seq.ipynb needs annotation or to be removed.

8. You should have a README.MD to explain the project and what each of the files represent.

File: code/extract_data.py (hunk 2 above)
Comment (Prajakta15Patil) [reply code/extract_data.py:97]: Added docstrings in the file

**Review by upomodoro — CHANGES_REQUESTED** [review]
So all the notebooks look much better. However, I actually would like to see more commentary both in the notebooks and the README. To give you a sense of a more appropriate level of annotation, I've provide a link to a notebook one of my other SharpestMind Mentees made: https://github.com/drewlehe/Nashville/blob/master/Analysis.ipynb.

Let me know if you have any questions.

File: code/extract_data.py (hunk 1 above)
Comment (Prajakta15Patil) [reply code/extract_data.py]: Removed the comment

**Review by upomodoro — APPROVED** [review]

**Review by upomodoro — APPROVED** [review]

**Pull request merged**